code/app/models.py

```diff
@@ -78,7 +78,46 @@ def password(self, password):
 
     def verify_password(self, password):
         return check_password_hash(self.password_hash, password)
-
+    
+    def generate_confirmation_token(self, expiration=30800):
+        """Generate a confirmation token to email a new user."""
+        s = Serializer(current_app.config['SECRET_KEY'], salt='email-confirm')
+        return s.dumps(self.id)
+
+    def confirm_account(self, token):
+        """Verify that the provided token is for this user's id."""
+        s = Serializer(current_app.config['SECRET_KEY'])
+        try:
+            email = s.loads(token, salt='email-confirm', max_age=60)
```

Comment:
```suggestion
            email = s.loads(token, salt='email-confirm', max_age=600)
```
previous setting of 60 sec was not enough time to receive email and click to confirm. Changed to 10 min, but prob should be longer for real users.